angr/storage/memory_mixins/paged_memory/pages/ultra_page.py

```diff
@@ -181,3 +296,42 @@ def changed_bytes(self, other, page_addr=None):
                             changes.add(addr)
 
         return changes
+
+    def contains(self, start: int, page_addr: int):
```

Comment:
can this be a private function?

Reply:
I feel we should support exposing a public `contains()` interface for inclusion tests.